delegation_survey/ph1_dre_adms_for_delegation.py

```diff
@@ -12,8 +12,8 @@
 '''
 LOGGER = Logger('ADM Converter')
 UPDATE_MONGO = True
-DELETE_MONGO = True
-EVAL = 5
+DELETE_MONGO = False
+EVAL = 4
```

Comment:
Just want to make sure that this is deliberately the new default.

Reply:
Yes, this was deliberate. This shouldn't have to run for its original purpose anymore because we have already generated the survey, but it definitely needs to run with the new script and these defaults help with that fix